api/src/main/java/eu/darkbot/api/managers/EntitiesAPI.java

```diff
@@ -164,6 +166,16 @@ public interface EntitiesAPI extends API.Singleton {
     @UnmodifiableView
     Collection<? extends Entity> getAll();
 
+    /**
+     * Creating a fake mine on a map that will be counted by pathfinder
+     */
+    FakeEntity.FakeMine createFakeMine(int typeId, Location loc, long removeDistance, long keepAlive);
+
+    /**
+     * Creating a fake box on a map
+     */
+    FakeEntity.FakeBox createFakeBox(String boxName, Location loc, long removeDistance, long keepAlive, boolean removeIfAttemptSelect);
+
```

Comment:
Missing usage for `FakeShip` and `FakeNpc` right? should probably have them here too

Reply:
FakeNpc, FakePlayer, FakePet classes (in main repo) should be implemented FakeShip interface in future. For each this implementation should be created separated method that return FakeShip. I not fully investigated behavior with ships (include plugins code) for correct implementation it so it is conceptual interface for future implementation